### PR TITLE
chore(deps): upgraded nestjs version to v8

### PR DESCRIPTION
**Pull request by Larinel**
Hi

Just upgraded nest js version to v8 to mitigate warning when using this package with 8.x version of nest

### CODE REVIEW

**Comment by Larinel** [comment]
@willsoto can you review this, please?

File: package.json [hunk 1]
```diff
@@ -137,7 +137,7 @@
     "typescript": "~4.3.5"
   },
   "peerDependencies": {
-    "@nestjs/common": "^7.0.0",
+    "@nestjs/common": "^7.0.0 || ^8.0.0",
```
Comment (willsoto) [inline package.json:140]: If you are updating everything to v8 then there is no need to keep `^7.0.0` here is there?

**Review by willsoto — CHANGES_REQUESTED** [review]
Since we are bumping to Nest v8, you will need to include the fact that this is a breaking change in your commit message so we can bump to the next major version

File: package.json (hunk 1 above)
Comment (Larinel) [reply package.json:140]: I tested with both v7v and v8 all tests are green, so actually, there is no need to drop support of v7 nest

Comment (willsoto) [reply package.json:140]: Then why upgrade the dependencies themselves? Just change the `peerDependencies`

Comment (Larinel) [reply package.json:140]: updated

**Comment by willsoto** [comment]
[Released](https://github.com/willsoto/nestjs-prometheus/blob/main/CHANGELOG.md#400-2021-07-15). Thanks

**Pull request merged**